packages/eslint-config/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@xt-pagesource/eslint-config",
```

Comment:
we need to change the names of the packages if we change the structure.

Because we may want to use the same repo for vanilla js, angular, vue etc. The package name should clearly signify the same

eg: xt-pagesource/eslint-config-react

Reply:
@ankurkakroo2  - Please have discussion with Vinod that was not the intent.

Reply:
ok.

Reply:
@ankurkakroo2 - I mean to say that was not the intent and ask to create different packages for each lib/framework.

Reply:
this comments is same as above comment for folder structure changes and move to react specific.